kiwi/tasks/system_build.py

```diff
@@ -189,7 +189,9 @@ def process(self):
         del manager
 
         # setup permanent image repositories after cleanup
-        setup.import_repositories_marked_as_imageinclude()
+        repositories = self.xml_state.get_repository_sections()
+        if bool(list(xr for xr in repositories if xr.get_imageinclude())):
```

Comment:
wouldn't it be nice to have this as a method in xml_state.py e.g

```python
def get_repositories_marked_as_imageinclude(self):
    repositories = self.get_repository_sections()
    return bool(list(repo for repo in repositories if repo.get_imageinclude()))
```

* That's what xml_state is for, statefull information about the current description.
* We would also avoid duplicating the code as now done in system_prepare